force_gromacs/data_sources/simulation/simulation_data_source.py

```diff
@@ -13,6 +16,8 @@ class SimulationDataSource(BaseDataSource):
     locally, or export the bash script in order to run on a remote
     cluster."""
 
+    simulation_builder = Instance(GromacsSimulationBuilder)
```

Comment:
Included the `simulation_builder` trait in order to refer to the attribute in decorated or overloaded `run` methods.